engine/src/object.h

```diff
@@ -374,7 +380,8 @@ class MCObject : public MCDLlist
 	
 	virtual void timer(MCNameRef mptr, MCParameter *params);
 	virtual uint2 gettransient() const;
-	virtual void setrect(const MCRectangle &nrect);
+	virtual void doSetRect(const MCRectangle &nrect);
```

Comment:
I'd suggest making this 'applyrect' rather than doSetRect, it is a little more consistent than the camel-case version.